include/menus/install_menu.sh

```
@@ -254,6 +279,10 @@ fi
           printf "E. QuattroBox Board Type : %s \n" "$qb_board_type"
           printf "F. QuattroBox Motor Type : %s \n" "$qb_motor_type"
           ;;
+        "EMU")
+          printf "G. EMU Lane Count : %s \n" "$emu_num_lanes"
```

Shellcheck (0.11.0)

[warning] emu_num_lanes is referenced but not assigned.

(SC2154)